volumegroup/README.md

```diff
@@ -0,0 +1,331 @@
+# CSI-Addons Operation: VolumeGroup
+
+based on deleted [3476-volume-group kep](https://github.com/kubernetes/enhancements/blob/590aa4eecb192d96e538020fc888459d9413221a/keps/sig-storage/3476-volume-group/README.md)
+
+## Container Storage Interface
+
+This section describes the interface between COs and Plugins.
```

Comment:
A description of what VolumeGroups are, and how users can benefit from them would be good to have here. Some hints are in the CreateVolumeGroup section, but more practical examples should be added here.

Reply:
done